api-conversion/README.md

```diff
@@ -66,6 +66,8 @@ Convert Custom Resources from YAML file
                           Creates an output YAML file for the converted custom resource
 ```
 
+**Once you converted the YAML files, apply the changes to Kubernetes using `kubectl apply -f` or `kubectl replace -f`.** 
```

Comment:
```suggestion
**When you have converted the YAML files, apply the changes to Kubernetes using `kubectl apply -f` or `kubectl replace -f`.** 
```

Reply:
Is the api-conversion.sh a _kubectl exec_?

Reply:
No, none of these is using `kubectl` directly. The `convert-resource` is using Kubernetes APIs directly. `convert-file` just outputs a file and the user has to deal with it (might be `kubectl apply`, but also just committing it to Git and waiting for some pipeline to deploy it etc.)

Reply:
Okay. Thanks

Reply:
I can't add comments to the convert-resource help lines. But, to make consistent with the convert-files:

Convert Custom Resources directly in Kubernetes
  -a, --all-namespaces   **Converts** resources in all namespaces
  -d, --debug            **Runs the tool in debug mode**
  -k, --kind[=<kinds> [<kinds> [<kinds> [<kinds> [<kinds> [<kinds> [<kinds>
        [<kinds> [<kinds> [<kinds>]]]]]]]]]]
                         **Specifies the kinds of custom resources to be converted, or converts all resources if not specified**
      -ll, --log-level=<level>
                         **Sets the log level to enable logging**
  -n, --namespace=<namespace>
                         **Specifies a Kubernetes namespace or OpenShift project, or uses the current namespace if not specified**
      --name=<name>      **If --namespace and a single custom resource --kind is used, specifies the name of the custom resource being converted**